php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -259,4 +260,19 @@ public static boolean isNullOrEmptyString(ExpressionTree tree) {
     return false;
   }
 
+  @CheckForNull
+  public static Tree getParentOfKind(Tree tree, Kind kind) {
```

Comment:
I just noticed that there's a somewhat similar method: `TreeUtils.findAncestorWithKind(Tree tree, Collection<Tree.Kind> kinds)`.